tasks/winbuildscripts/Generate-Chocolatey-Package.ps1

```diff
@@ -54,4 +62,9 @@ if (!(Test-Path $outputDirectory)) {
     New-Item -ItemType Directory -Path $outputDirectory
 }
 
+if ($installMethod -eq "online") {
+    # Set the $url in the install script
+    (Get-Content $installScript).replace('$url', $url) | Set-Content $installScript
```

Comment:
I would add the quotes around the URL here, before it gets replaced, instead of when it's defined (1 place instead of 2, and makes the intent more clear). And maybe we can replace something like `###_URL_TO_BE_REPLACED_HERE_###` instead of `$url` which might look like a regular variable to someone reading/modifying the script?